perfkitbenchmarker/benchmarks/fio_benchmark.py

```diff
@@ -90,11 +91,17 @@
                      'The size of the working set, in GB. If not given, use '
                      'the full size of the device.',
                      lower_bound=0)
+flags.DEFINE_integer('run_for_minutes', 10,
+                     'Repeat the job scenario(s) for the given number of '
+                     'minutes. Only valid when using --generate_scenarios. '
+                     'When using multiple scenarios, each one is run for the '
+                     'given number of minutes.',
+                     lower_bound=0)
```

Comment:
If you set it to run for 0 minutes, it should do a dry run. Everything looks exactly like a run, except that it never runs the fio command (and never tries to parse the results). I made this 0 because the behavior at 0 fits the pattern that happens at positive integers, so I didn't feel like I had a reason to forbid it. I can make it 1 - I don't have strong feelings about this.